hera_qm/utils.py

```diff
@@ -32,361 +33,381 @@ def _str_to_bytes(s):
+                        'These are all thrown away at once without waiting to iteratively throw away only the worst offender.')
+        ap.add_argument('--extension', default='.ant_metrics.json', type=str,
+                        help='Extension to be appended to the file name. Default is ".ant_metrics.json"')
+        ap.add_argument('--metrics_path', default='', type=str,
+                        help='Path to save metrics file to. Default is same directory as file.')
+        ap.add_argument('--vis_format', default='miriad', type=str,
+                        help='File format for visibility files. Default is miriad.')
+        ap.add_argument('-q', '--quiet', action='store_false', dest='verbose', default=True,
+                        help='Silence feedback to the command line.')
+        ap.add_argument('files', metavar='files', type=str, nargs='*', default=[],
+                        help='*.uv files for which to calculate ant_metrics.')
+
+        ap.add_argument('--run_mean_vij', action='store_true',
+                        dest='run_mean_vij',
+                        help=('Sets boolean flag to True. Flag determines if '
+                              'mean_vij_metrics is run. Default: True'))
+        ap.add_argument('--skip_mean_vij', action='store_false',
+                        dest='run_mean_vij',
+                        help=('Sets boolean flag to False. Flag determines if '
+                              'mean_vij_metrics is run. Default: True'))
+        ap.set_defaults(run_mean_vij=True)
+
+        ap.add_argument('--run_red_corr', action='store_true',
+                        dest='run_red_corr',
+                        help=('Sets boolean flag to True. Flag determines if '
+                              'red_corr_metrics is run. Default: True'))
+        ap.add_argument('--skip_red_corr', action='store_false',
+                        dest='run_red_corr',
+                        help=('Sets boolean flag to False. Flag determines if '
+                              'red_corr_metrics is run. Default: True'))
+        ap.set_defaults(run_red_corr=True)
+
+        ap.add_argument('--run_cross_pols', action='store_true',
+                        dest='run_cross_pols',
+                        help=('Sets boolean flag to True. Flag determines if '
+                              'mean_Vij_cross_pol_metrics and '
+                              'red_corr_cross_pol_metrics are run. '
+                              'Default: True'))
+        ap.add_argument('--skip_cross_pols', action='store_false',
+                        dest='run_cross_pols',
+                        help=('Sets boolean flag to False. Flag determines if '
+                              'mean_Vij_cross_pol_metrics and '
+                              'red_corr_cross_pol_metrics are run. '
+                              'Default: True'))
+        ap.set_defaults(run_cross_pols=True)
```

Comment:
YAY :tada: for no single letter variables

Reply:
hey now, there are still plenty of single letter variables.